src/common/VirtualTerminal.h

```diff
@@ -20,8 +20,11 @@
 #ifndef SDDM_VIRTUALTERMINAL_H
 #define SDDM_VIRTUALTERMINAL_H
 
+#include <QString>
+
 namespace SDDM {
     namespace VirtualTerminal {
+        QString getVtPath(int vt);
```

Comment:
Can we do away without the get prefix? we are not using it. Also it's in the VirtualTerminal namespace, we don't need to reiterate (we do reiterate on other cases, but it's not something we need to perpetuate).
```suggestion
        QString path(int vt);
```